docs/source/en/model_doc/mvp.mdx

```diff
@@ -0,0 +1,142 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# MVP
+
+## Overview
+
+The MVP model was proposed in [MVP: Multi-task Supervised Pre-training for Natural Language Generation](https://github.com/RUCAIBox/MVP) by Tianyi Tang, Junyi Li, Wayne Xin Zhao and Ji-Rong Wen.
```

Comment:
Is there no paper published alongside the model?

Reply:
Or will there be a paper in some days / weeks? :-)

Reply:
We have uploaded our paper to ArXiv, and it will be published in June 27. We have also uploaded it in [https://github.com/RUCAIBox/MVP/blob/main/paper.pdf](https://github.com/RUCAIBox/MVP/blob/main/paper.pdf).